src/test/resources/telemetry.properties

```diff
@@ -0,0 +1,13 @@
+# number of second to check if it should send a message
+telemetry.daemon.thread.loop.seconds=10
+# remote data collector server base url
+telemetry-gateway-service.base-url=http://localhost:8080 
+
+# enabled remote geo ip lookup service
+# allowed values:
+# - geoiplookup-remote-service
+# - ipapi-remote-service
+telemetry.enabled.geo.ip.lookup.service=geoiplookup-remote-service
+# endpoints remote services
+geoiplookup-remote-service.ribbon.base-url=https://json.geoiplookup.io
+ipapi-remote-service.ribbon.base-url=http://ip-api.com/json 
```

Comment:
Should this be `https`?

Reply:
free API don't support SSL https://ip-api.com/docs/api:json